CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+- Fix: Alarm for notification queue overpassing a given threshold (#4113)
```

Comment:
```suggestion
- Fix: alarm for notification queue overpassing a given threshold (#4113)
```

Reply:
Updated in https://github.com/telefonicaid/fiware-orion/pull/4375/commits/51a71bbd8113d91b0c1858d8f1f1aada294a61be